SwiftlyUserDefault/SwiftlyUserDefault/SwiftlyUserDefault.swift

```diff
@@ -20,23 +21,21 @@ public class SwiftlyUserDefault: NSObject {
     
     @discardableResult
     public static func setValue<T>(_ key: SwiftlyUserDefaultable, value: T) -> Bool {
-        db.setValue(value, forKey: key.key)
-        return db.synchronize()
+        db.setValue(key, value: value)
+        return true
```

Comment:
This exercise was all about maintainability. Even though `synchronize()` should not be used anymore because it will soon be deprecated, we don't want to change the signature of this function. Hence, the `return true`.